integrationTests/singleShard/smartContract/dns_test.go

```diff
@@ -0,0 +1,33 @@
+package smartContract
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm/arwen"
+	"github.com/ElrondNetwork/elrond-go/process"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDNS_Register(t *testing.T) {
```

Comment:
add a test for a query when the username is already set for one account

Reply:
Added test (single-shard).